packages/docs-app/src/styles/_examples.scss

```diff
@@ -711,6 +711,20 @@ $docs-hotkey-piano-height: 510px;
   }
 }
 
+#{example("MultiSelect")} .docs-example {
+  .docs-custom-target {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    min-width: 145px;
```

Comment:
could we override the `rightPadding` here so the radius of the number tag lines up with the edge of the target? (guessing this docs example may end up being copied)

![Screenshot 2024-07-18 at 10 09 57 AM](https://github.com/user-attachments/assets/0c44a70b-2863-48e4-9b3e-05ba2b59329a)